browser/layout/wrapper.html

```diff
@@ -132,13 +132,15 @@
 				{{{body}}}
 			</div>
 
+			{{#unlessEquals nUi.footer.hide true}}
```

Comment:
If this is always a boolean, `{{#unlessEquals nUi.footer.hide}}` would suffice.

Reply:
it's true.. but I have seen in some codebases the use of 'false' as a string for example, I just wanted to make this strictly typed and bulletproof as it has a major impact

Reply:
does footer have any other options?

Reply:
It seems odd to have a `hide` property as an option.

Reply:
I did consider just nUi.footer. I figured using the same structure as nUi.header made sense - less for a dev to consider, and leaves room for new options.
Open to alternative suggestions to hide property

Reply:
I think we should mark this up to improve in v9

Reply:
what's v9 sorry?

Reply:
Sorry, the next major version. I added it to #1229